### PR TITLE
Added an HTTP2 server example

### PR DESCRIPTION
**Pull request by shahzeb1**
As requested in this issue: https://github.com/zeit/next.js/issues/6850

This is a basic HTTP2 server which can handle the `/` and `/about` routes. 

The Readme includes [specific instruction](https://github.com/shahzeb1/next.js/blob/31877720f076b49b39a167deb0ba8650d0052dda/examples/with-http2/README.md#download-manually) about how to generate the public/private keys used for this demo server.

### CODE REVIEW

File: examples/with-http2/package.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/zeit/next.js/"
+  },
+  "keywords": [
+    "http2",
+    "next"
+  ],
+  "author": "Shahzeb Khan <https://github.com/shahzeb1>",
```
Comment (timneutkens) [inline examples/with-http2/package.json]: Can you make sure the package.json matches other examples? We generally leave out the author, repository, add in a license etc.

**Review by timneutkens — CHANGES_REQUESTED** [review]
Added one small comment

**Comment by shahzeb1** [comment]
Went ahead and made the `package.json` match the one located in the `hello-world` example.

**Comment by shahzeb1** [comment]
Also as far as I can tell, the ziet.next.js test that just failed doesn't have to do with my commits. I'm not sure why it would fail this PR.

**Review by timneutkens — APPROVED** [review]
Awesome! Thank you!

**Pull request merged**